meteo_lt/models.py

```diff
@@ -51,6 +63,7 @@ class ForecastTimestamp:
     pressure: float = field(metadata={"json_key": "seaLevelPressure"})
     humidity: float = field(metadata={"json_key": "relativeHumidity"})
     precipitation: float = field(metadata={"json_key": "totalPrecipitation"})
+    warnings: List["WeatherWarning"] = field(default_factory=list, init=False)
```

Comment:
```suggestion
    warnings: List[WeatherWarning] = field(default_factory=list, init=False)
```

and move `WeatherWarning` class above this one